app/javascript/components/responseTypes/TextResponse.js

```diff
@@ -0,0 +1,31 @@
+
+import * as React from 'react';
+import TextField from '@mui/material/TextField';
+import { Typography } from '@mui/material';
+
+class TextResponse extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const {
+            question
+        } = this.props;
+        return ( 
+            <div>
+                <Typography variant="h5">{question}</Typography>
+                <TextField
```

Comment:
I guess my main question here is how do we store the response so it can be sent later, is the response stored in the this text response component, or in the parent Survey.js component.